flink-yarn/src/main/java/org/apache/flink/yarn/Utils.java

```diff
@@ -263,6 +265,13 @@ private static LocalResource registerLocalResource(FileSystem fs, Path remoteRsr
 		return localResource;
 	}
 
+	public static void setAclsFor(ContainerLaunchContext amContainer, org.apache.flink.configuration.Configuration flinkConfig) {
+		amContainer.setApplicationACLs(new HashMap<ApplicationAccessType, String>(){{
```

Comment:
I would feel more comfortable if we only set `ApplicationAccessType.VIEW_APP` and `ApplicationAccessType.MODIFY_APP`, if the user actually configured application ACLs. Also, can we make the default values of the new config options `null`? What do you think?